src/EnergyPlus/AirflowNetwork/src/Solver.cpp

```diff
@@ -1172,16 +1172,20 @@ namespace AirflowNetwork {
                     SpecifiedMassFlowData.emplace_back();
                     SpecifiedMassFlowData[i_mass].name = thisObjectName;
                     SpecifiedMassFlowData[i_mass].mass_flow = flow_rate;
-                    elements[thisObjectName] = &SpecifiedMassFlowData[i_mass]; // Yet another workaround
                     ++i_mass;
                 } else {
                     SpecifiedVolumeFlowData.emplace_back();
                     SpecifiedVolumeFlowData[i_vol].name = thisObjectName;
                     SpecifiedVolumeFlowData[i_vol].volume_flow = flow_rate;
-                    elements[thisObjectName] = &SpecifiedVolumeFlowData[i_vol]; // Yet another workaround
                     ++i_vol;
                 }
             }
+            for (auto &afe : SpecifiedMassFlowData) {
+                elements[afe.name] = &afe; // Yet another workaround
+            }
+            for (auto &afe : SpecifiedVolumeFlowData) {
+                elements[afe.name] = &afe; // Yet another workaround
+            }
```

Comment:
Unfortunately I was taking the address before I was done filling out the arrays, and that's bad. This changes the order that things are done in so that the address taking happens after all the vectors are filled in.

Reply:
Makes perfect sense.  I tried running the defect file but even in develop it succeeded.  It was a release build so I assume it was just passing through with weird data.  Anyway, this fix is super clear and I'm going to get it in.